fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/mixin/itemgroup/client/MixinCreativePlayerInventoryGui.java

```diff
@@ -49,12 +50,12 @@ public MixinCreativePlayerInventoryGui(Container container_1, PlayerInventory pl
 
 	private int fabric_getPageOffset(int page) {
 		switch (page) {
-			case 0:
-				return 0;
-			case 1:
-				return 12;
-			default:
-				return 12 + ((12 - FabricCreativeGuiComponents.COMMON_GROUPS.size()) * (page - 1));
+		case 0:
```

Comment:
Im not too sure about cases being like this. Im they should be indented from the switch?

Reply:
they are labels, labels are typically not being indented, making the effective indentation level of the switch expression one level.

Reply:
Where does that leave the case where the labels come with blocks:
```java
switch (thing) {
	case 5: { //Blocks get indented
		String name = "123";
		...
	}

case 4: //Labels don't
	return "150";

	case 3: {
		String name = "321";
		...
	}
}
```
I guess in such scenarios blocky switches could be indented whilst unblocky ones wouldn't, although that's not very consistent.

Reply:
They don't get indented either, similar to how `if {<whatever>}` only indents `<whatever>`, not the entire if statement. This obviously has the side effect where two consecutive `}` end up on the same level, which may look a little odd.and Eclipse.

Note that this is a very common/standard convention with some previous uses in Fabric-API and IMO it's also more adequate since it better reflects how case labels work much more like goto target labels than isolated/leaf handler blocks.